lineages/B.D.E.1.7.yml

```diff
@@ -0,0 +1,18 @@
+defining_mutations:
+  - locus: G
+    position: 32
+    state: R
+  - locus: G
+    position: 223
+    state: E

```

Comment:
In addition to these, I see G:T139I and L:S1320P (and some other G's that are in a position range where we wouldn't match due to my use of the old RefSeq as reference).